packages/fxa-auth-server/package.json

```diff
@@ -40,6 +40,7 @@
     "test": "VERIFIER_VERSION=0 scripts/test-local.sh",
     "test-unit": "VERIFIER_VERSION=0 TEST_TYPE=unit scripts/test-ci.sh",
     "test-integration": "VERIFIER_VERSION=0 TEST_TYPE=integration scripts/test-ci.sh",
+    "test-integration-v2": "VERIFIER_VERSION=0 TEST_TYPE=integration-v2 scripts/test-ci.sh",
```

Comment:
Should this be `VERIFIER_VERSION=1`?

Reply:
No! The verifier version is not changing. As I dug into this, it became apparent this was only designed to change the server side stretch, and wasn't intended for client side stretch changes.